### PR TITLE
rax modules: deprecation

### PR DESCRIPTION
**Pull request by russoz**
##### SUMMARY
Deprecate rax modules

##### ISSUE TYPE
- Bugfix Pull Request
- Feature Pull Request

##### COMPONENT NAME
plugins/modules/rax_cbs_attachments.py
plugins/modules/rax_cbs.py
plugins/modules/rax_cdb_database.py
plugins/modules/rax_cdb.py
plugins/modules/rax_cdb_user.py
plugins/modules/rax_clb_nodes.py
plugins/modules/rax_clb.py
plugins/modules/rax_clb_ssl.py
plugins/modules/rax_dns.py
plugins/modules/rax_dns_record.py
plugins/modules/rax_facts.py
plugins/modules/rax_files_objects.py
plugins/modules/rax_files.py
plugins/modules/rax_identity.py
plugins/modules/rax_keypair.py
plugins/modules/rax_meta.py
plugins/modules/rax_mon_alarm.py
plugins/modules/rax_mon_check.py
plugins/modules/rax_mon_entity.py
plugins/modules/rax_mon_notification_plan.py
plugins/modules/rax_mon_notification.py
plugins/modules/rax_network.py
plugins/modules/rax.py
plugins/modules/rax_queue.py
plugins/modules/rax_scaling_group.py
plugins/modules/rax_scaling_policy.py

### CODE REVIEW

**Comment by felixfontein** [comment]
This needs a changelog fragment as well.

**Comment by russoz** [comment]
> This needs a changelog fragment as well.

I am waiting for the CI to run, and it looks like it will need a re-run on a couple of tests.

File: plugins/doc_fragments/rackspace.py [hunk 1]
```diff
@@ -105,6 +105,10 @@ class ModuleDocFragment(object):
       - Whether or not to require SSL validation of API endpoints.
     type: bool
     aliases: [ verify_ssl ]
+deprecated:
+  removed_in: 9.0.0
+  why: This module relies on the deprecated package pyrax.
+  alternative: Use the Openstack modules isntead.
```
Comment (felixfontein) [inline plugins/doc_fragments/rackspace.py:111]:
```suggestion
  alternative: Use the Openstack modules instead.
```

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@russoz thanks!

**Pull request merged**